src/engines/bibtex.rs

```diff
@@ -9,6 +9,12 @@ use super::{ExecutionState, IoEventBackend, TectonicBridgeApi};
 use crate::errors::{ErrorKind, Result};
 use crate::io::IoStack;
 use crate::status::StatusBackend;
+use crate::unstable_opts::UnstableOptions;
+
+#[no_mangle]
+extern "C" {
+    static mut min_crossrefs: i32;
+}
```

Comment:
I have been trying to minimize the number of direct symbol references with `extern "C"` names and such. Thus far, I've wanted to control these kinds of options through APIs such as `tt_xetex_set_int_variable`. One approach that I'd prefer here would be to add a `tt_bibtex_set_int_variable` here.

Another approach that is now possible would be to start moving this kind of state to the Rust side, and expose it to the C through the new cbindgen support. There could be a `tectonic_get_min_crossrefs()` function or something along those lines. This is appealing to me since it keeps more of the control logic on the safe Rust side.

Would it be too much of a pain to try one of these approaches instead of the direct variable linkage?